tests/Fixtures/TestController.php

```diff
@@ -40,6 +40,9 @@ public function withGroupOverride()
      * @bodyParam another_one number Just need something here.
      * @bodyParam yet_another_param object required
      * @bodyParam even_more_param array
+     * @bodyParam book.name string
```

Comment:
What happens if they write `@bodyParam book object` in addition to these? Also, how about arrays?

Reply:
As for examples, they work the same way as before. If `book.name` ends with `Example: The New Dawn`, then the result would be `{"book": {"name": "The New Dawn", ...}, ...}`.